templates/includes/footer.html

```diff
@@ -11,22 +11,22 @@
         <nav class="tna-footer__legal tna-column tna-column--full" aria-label="General site links">
             <ul class="tna-footer__legal-items tna-ul tna-ul--plain">
                 <li class="tna-footer__legal-item">
-                    <a href="https://www.nationalarchives.gov.uk/help/web-accessibility/" class="tna-footer__legal-item-link">
+                    <a href="https://ohos-zgoyvpy-rasrzs7pi6sd4.uk-1.platformsh.site/using-our-website/#h2.accessibility-statement" class="tna-footer__legal-item-link">
```

Comment:
```suggestion
                    <a href="/using-our-website/#h2.accessibility-statement" class="tna-footer__legal-item-link">
```
Would a relative link be better?

Reply:
I agree. Is there an easy wayto get the base url via python? Something like `{% sitebase %} `

Reply:
It would get resolved on its own. 
see example [vis_view_img="/static/images/ohos-homepage-maps.jpg"](https://github.com/nationalarchives/ds-ohos-wagtail/blob/develop/templates/home/home_page.html#L40-L44)

Reply:
hmmn, well http://127.0.0.1:8000/using-our-website/#h2.accessibility-statement for example gives me a 404.

Reply:
The user-our-website page is tied to a login.

Reply:
The login can removed via Wagtail Admin as required.